lnwallet/channel.go

```diff
@@ -1301,6 +1301,7 @@ func (lc *LightningChannel) fetchCommitmentView(remoteChain bool,
 	//  * when dumping fees back into initiator output, only dumb explicit
 	//    fee
 	var dustLimit btcutil.Amount
+	var dustFees btcutil.Amount
```

Comment:
Minor style nit: within the codebase whenever we have two or more variable declarations following each other we collapse them into a single `var` declaration like so: 
```go
var (
    dustLimit btcutil.Amount
    dustFees
)
```

Alternatively, since they're of the same type, we can do something like this: 
```go
var dustLimit, dustFees btcutil.Amount
```